packages/data-point/lib/entity-types/factory.test.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-env jest */
+
+const { createEntityFactory } = require('./factory')
+
+describe('creating a model with createEntityFactory', () => {
+  const factory = createEntityFactory('model')
```

Comment:
place this inside a beforeAll otherwise it gets executed before all tests in DataPoint suite

Reply:
ohhh did not realize that!